streaming_jupyter_integrations/magics.py

```diff
@@ -369,6 +359,25 @@ def __enrich_cell(self, cell: str) -> str:
         joined_cell = inline_sql_in_cell(enriched_cell)
         return joined_cell
 
+    def __flink_execute_sql_file(self, path: str) -> None:
+        if self.background_execution_in_progress:
+            self.__retract_user_as_something_is_executing_in_background()
+            return
+
+        if os.path.exists(path):
+            with open(path, "r") as f:
+                statements = map(lambda s: self.__enrich_cell(s.rstrip(';')), sqlparse.split(f.read()))
```

Comment:
Maybe we should use for-comprehension instead of `map`?
```suggestion
                statements = [self.__enrich_cell(s.rstrip(';')) for s in sqlparse.split(f.read())]
```
---
BTW, in `.pre-commit-config.yaml`, there's an argument to flake8: `--exclude=streaming_jupyter_integrations/magics.py`. I'd rather have it removed, so flake would point to this line with a similar comment. I guess this exclude ended up there because of line 55 in `magics.py` file (the one starting with `get_getaway().jvm.org...`), however the fix to this is to put `  # noqa: E501` at the end of the line, instead of disabling the checks at all:
```python
self.s_env = StreamExecutionEnvironment(
    get_gateway().jvm.org.apache.flink.streaming.api.environment.StreamExecutionEnvironment.createLocalEnvironmentWithWebUI(  # noqa: E501
        conf._j_configuration
    )
)
```